src/content/docs/en/guides/integrations-guide/mdx.mdx

```diff
@@ -326,13 +330,25 @@ export default defineConfig({
 
 ### `recmaPlugins`
 
+<p>
+
+**Type:** `PluggableList`<br />
```

Comment:
> But for mdx I don't know who we could ask... I hesitated to use unified.PluggableList, but I wasn't sure this would be more explicit for people.

Let's ask @delucis !

(Armand's concern is regarding the entry for `recmaPlugins`: "PluggableList is not an Astro type but a unified one and I don't think we reexport it, but I don't have a better idea to describe the type.)